blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/ScmResourceImpl.java

```diff
@@ -59,7 +60,12 @@ public Link getLink() {
     }
 
     private @Nonnull User checkPermission(){
-        ACL acl = item.getACL();
+        ACL acl;
+        if(item.getParent() != null && item.getParent() instanceof OrganizationFolder){
```

Comment:
Why would the `item` `ACL` not allow something that the `item.parent` would?

Reply:
Because in this case we are inside an OrganizationFolder which have computed folders that you can't configure.